### ISSUE TITLE
[a11y] Some hidden UI element are still visible to the screen reader

### ISSUE THREAD

**Issue by bmarty** [vector-im/element-call#3384]
### Steps to reproduce

1. On an Android device enable the screen reader
2. Run Element X Android (there is maybe the same problem on iOS, I did not check)
3. Start a Call
4. Navigate using the accessibility gesture through the screen

#### What did you expect?

Invisible element are not focusable by the screen reader

#### What happened instead?

hidden element can be focused by the screen reader.

<img width="418" alt="Image" src="https://github.com/user-attachments/assets/effa6e73-80aa-4c63-a63a-d00933fa3a24" />

<img width="433" alt="Image" src="https://github.com/user-attachments/assets/480cb956-eeac-4d2b-b844-70e0a0525a72" />

<img width="404" alt="Image" src="https://github.com/user-attachments/assets/c3aa832a-40ed-484b-bb4b-16d8683ffbab" />

And others:
"Spotlight"

<img width="406" alt="Image" src="https://github.com/user-attachments/assets/ce561207-fd0b-415b-94f3-1ef96bd69118" />

"Grid"

<img width="407" alt="Image" src="https://github.com/user-attachments/assets/ec2e038d-8ed4-4f1c-bc14-d2623182c6af" />

6. Additionally a label is missing for the mute icon

<img width="79" alt="Image" src="https://github.com/user-attachments/assets/ca40e674-a43a-4add-b42b-0cacf054cf03" />

7. And when opening the settings, the focus does not change and user can still navigate on the item below:

<img width="398" alt="Image" src="https://github.com/user-attachments/assets/ad226345-5dec-4603-acb8-15b910fadd85" />

### Operating system

Android (and maybe iOS and web)

### Will you send logs?

No
